src/sagemaker/predictor.py

```diff
@@ -230,12 +230,13 @@ def _row_to_csv(obj):
 
 
 class _CsvDeserializer(object):
-    def __init__(self):
-        self.content_type = CONTENT_TYPE_CSV
+    def __init__(self, encoding='utf-8'):
```

Comment:
do you need to pass the accept in as well?

Reply:
I was staying consistent with:

```python
class _CsvSerializer(object):
    def __init__(self):
        self.content_type = CONTENT_TYPE_CSV
```

open to changing it though if there's a compelling use case